### ISSUE TITLE
Make create_graph disabling optional

### ISSUE THREAD

**Issue by OMalenfantThuot** [atomistic-machine-learning/schnetpack#279]
Hi, hope you're all doing well.

Since the last major [update](https://github.com/atomistic-machine-learning/schnetpack/pull/262), I had a few errors popping up in my codes, but I managed to solve everything today.
There is one thing that would help me a lot tough:
In `src/schnetpack/atomistic/output_modules.py` there is a new line in the `forward()` method:
```
create_graph = self.create_graph if self.training else False
```
This is problematic when using custom layers (like dropout layers) that needs a model.eval() but for which I still want to be able to use autograd.
Is there any way of making this optional?
If not, I'll try to find a workaround.

Thanks a lot!

**Comment by Stefaanhess** [comment]
Hi @OMalenfantThuot ,

I just changed the behaviour of `create_graph` in #280 . Now the graph is always created in training mode, if forces are required. If you also want to create the graph in eval mode, just set `create_graph=True` in the `__init__`. Does this solve your problem?

Best,
Stefaan

**Comment by OMalenfantThuot** [comment]
Hi @Stefaanhess .
Thanks for the quick answer. This is a perfect solution!

Olivier
